pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned_test.go

```diff
@@ -160,34 +162,74 @@ var _ = Describe("Tuned", func() {
 					bootLoader, err := tunedData.GetSection("bootloader")
 					Expect(err).ToNot(HaveOccurred())
 					Expect(bootLoader.Key("cmdline_power_performance").String()).To(Equal(cmdlineHighPowerConsumption))
+					Expect(bootLoader.Key("cmdline_pstate").String()).To(Equal(cmdlineHighPowerConsumptionPstate))
 				})
 			})
 
 			When("realtime workload enabled", func() {
 				It("should not contain idle=poll cmdline", func() {
 					profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
-						RealTime:             pointer.BoolPtr(false),
-						HighPowerConsumption: pointer.BoolPtr(false),
+						HighPowerConsumption: pointer.BoolPtr(true),
+						RealTime:             pointer.BoolPtr(true),
 					}
 					tunedData := getTunedStructuredData(profile)
 					bootLoader, err := tunedData.GetSection("bootloader")
 					Expect(err).ToNot(HaveOccurred())
-					Expect(bootLoader.Key("cmdline_idle_poll").String()).ToNot(Equal(cmdlineIdlePoll))
+					Expect(bootLoader.Key("cmdline_idle_poll").String()).To(Equal(cmdlineIdlePoll))
 				})
 			})
 
 			When("realtime workload disabled", func() {
 				It("should contain idle=poll cmdline", func() {
 					profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
 						HighPowerConsumption: pointer.BoolPtr(true),
-						RealTime:             pointer.BoolPtr(true),
+						RealTime:             pointer.BoolPtr(false),
 					}
 					tunedData := getTunedStructuredData(profile)
 					bootLoader, err := tunedData.GetSection("bootloader")
 					Expect(err).ToNot(HaveOccurred())
-					Expect(bootLoader.Key("cmdline_idle_poll").String()).To(Equal(cmdlineIdlePoll))
+					Expect(bootLoader.Key("cmdline_idle_poll").String()).ToNot(Equal(cmdlineIdlePoll))
 				})
 			})
+
+			When("perPodPowerManagement Hint to true", func() {
+				It("should fail as PerPodPowerManagement and HighPowerConsumption can not be set to true", func() {
+					profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
+						HighPowerConsumption:  pointer.BoolPtr(true),
+						PerPodPowerManagement: pointer.BoolPtr(true),
+					}
+					_, err := NewNodePerformance(profile)
+					Expect(err.Error()).To(ContainSubstring("Invalid WorkloadHints configuration: HighPowerConsumption is true and PerPodPowerManagement is true"))
```

Comment:
Do we have a validation hook that checks this behavior ? where do we prevent the user from setting both hints as true ?  should probably go under https://github.com/openshift/cluster-node-tuning-operator/blob/master/pkg/apis/performanceprofile/v2/performanceprofile_validation.go